pylint/checkers/utils.py

```diff
@@ -104,15 +105,15 @@
 SPECIAL_METHODS_PARAMS = {
     name: params
     for params, methods in _SPECIAL_METHODS_PARAMS.items()
-    for name in methods
+    for name in methods # type: ignore
```

Comment:
Ditto. what does this ignore, the iteration or the result of the dictionary creation?

Reply:
checkers/utils.py:108: error: "object" has no attribute "__iter__"; maybe "__str__" or "__dir__"? (not iterable)
somehow mypy not able to infer that method is a tuple. annotating _SPECIAL_METHODS_PARAMS might have fixed this, but didn't try it.